### PR TITLE
Fixed Redirecting Between Sorted Places Page And Places Page

### PR DESCRIPTION
**Pull request by Viktor-Hordynskyi**

### CODE REVIEW

**Review by andrii-kondratiuk — APPROVED** [review]
Nice job

File: src/components/pages/places-list/places-list.js [hunk 1]
```diff
@@ -29,95 +29,111 @@ export default class PlacesList extends Component {
...
 
-        this.setState({places: filteredPlaces});
-      } else {
-        const places = await PlacesApi.getAllPlaces();
-        const filteredPlaces = places.filter((place) => {
-          return place.isModerated === true &&  place.approved === true
-        })
+      this.setState({ places: filteredPlaces });
+    } catch (error) {
+      console.log("Handle loading all places error: ", error);
```
Comment (Dmitry528) [inline src/components/pages/places-list/places-list.js]: Console

Comment (Viktor-Hordynskyi) [reply src/components/pages/places-list/places-list.js]: Done

**Review by Dmitry528 — APPROVED** [review]
Good work!

**Review by marrybbb — APPROVED** [review]

**Pull request merged**